python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -1026,6 +1026,7 @@ class BuiltinDataset(OnDiskDataset):
         "ogbn-papers100M-seeds",
         "igb-hom-medium",
         "igb-hom-medium-seeds",
+        "igb-hom-large-seeds",
```

Comment:
@frozenbugs When we add a new dataset, do we still need to have both seeds version and nonseeds version?